src/lib/bitcoin/BitcoinUtils.js

```diff
@@ -114,4 +116,34 @@ class BitcoinUtils { // eslint-disable-line no-unused-vars
     static satoshisToCoins(satoshis) {
         return satoshis / BitcoinConstants.SATOSHIS_PER_COIN;
     }
+
+    /**
+     * @param {string | Uint8Array | Buffer} bytes
+     * @param {'BIP49' | 'BIP84'} bip
+     * @returns {string}
+     */
+    static addressBytesToAddress(bytes, bip) {
```

Comment:
This is a left-over from when I did BTC HTLC validation in the Keyguard before the change to the two-step process.

Will probably be re-used once HTLC data is validated again.